docs/contribute.rst

```diff
@@ -13,28 +13,79 @@ Start Developing
 ----------------
 
 To get started with your contribution, fork this project into your own GitHub profile and clone that forked repository to your machine.
-After cloning, navigate into the new directory and set the original project repository as the remote named ``upstream``.
+
+The path to your new repository for the ``git clone`` command will differ depending on whether you are using a password to authenticate to GitHub, or SSH keys.
+(If you did not set up SSH keys for your GitHub account, then you are using a password.)
+To find the correct path, go to your new repository's landing page, and click the green button that says "Clone or download".
+Copy the path it shows, and use this as the argument to the ``git clone`` command.
+
+The password version will look like (with ``your-username`` in place of your actual GitHub username):
+
+::
+
+    git clone https://github.com/your-username/django-graph-api.git
+
+The SSH keys version will look like:
+
+::
+
+    git clone git@github.com:your-username/django-graph-api.git
+
+After cloning, navigate into the new directory.
```

Comment:
I agree with @ariannedee that this detail is more than we want in the docs. If we're concerned about people not knowing how to fork or clone a repository using github, we should be directing them to Github docs on the subject. (But it would be good to have that as a separate PR.)